lms/static/js/instructor_dashboard/util.js

```diff
@@ -379,7 +379,7 @@
                 url: listEndpoint,
                 success: function(data) {
                     if (data.tasks.length) {
-                        ths.createTaskListTable(ths.$table_running_tasks, data.tasks);
+                        createTaskListTable(ths.$table_running_tasks, data.tasks);
```

Comment:
No, this refers to PendingInstructorTasks. I reverted line 529 to back it was in coffee.